client/src/components/MenuHeader.js

```diff
@@ -1,48 +1,109 @@
-						Sign In
-					</Menu.Item>
-		    </Menu.Menu>
-		  </Menu>
+  		<Grid>
+				<Grid.Row only='computer'>
+					<Grid.Column>
+				  	<Menu color='blue' inverted pointing secondary>
+					    <Menu.Item name='Fantasy Fitness' />
+							<MenuItemsRouting activeItem={activeItem} handleItemClick={this.handleItemClick} />	
+						</Menu>	
+						<RoutingPaths />
+					</Grid.Column>					
+				</Grid.Row>
+			  <Grid.Row only='tablet mobile'>
+			  	<Grid.Column>
+			  		<Menu color='blue' inverted>
+			  			<Menu.Item name='Fantasy Fitness' />
+			  			<Menu.Menu position='right'>
+			  				<Menu.Item onClick={this.toggleVisibility}>
+			  					<Icon name='sidebar' />
+			  				</Menu.Item>
+			  			</Menu.Menu>
+			  		</Menu>
+		  			<Sidebar.Pushable as={Segment}>
+		  				<Sidebar as={Menu} animation='overlay' width='wide' visible={visible} vertical inverted>
+								<MenuItemsRouting activeItem={activeItem} handleItemClick={this.handleItemClick} style={{padding: '20px', fontSize: '1em'}}/>		
+		  				</Sidebar>
+		  				<Sidebar.Pusher dimmed={dimmed}>
+		  					<Segment>
+		  						<RoutingPaths />
+								</Segment> 			
+		  				</Sidebar.Pusher>
+		  			</Sidebar.Pushable>			  		
+			  	</Grid.Column>
+			  </Grid.Row>
+		  </Grid>
   	)
   }
 }
 
+const MenuItemsRouting = ({activeItem, handleItemClick, style}) => (
+	<Menu.Menu position='right'>	
+		<Menu.Item to='/score' name='score form' active={activeItem === 'score form'} onClick={handleItemClick} style={style}>
+			Score
+		</Menu.Item>
+		<Menu.Item to='/scoresheet' name='score sheet' active={activeItem === 'score sheet'} onClick={handleItemClick} style={style}>
+			Score Sheet
+		</Menu.Item>
+		<Menu.Item to='/league' name='league' active={activeItem === 'league'} onClick={handleItemClick} style={style}>
+			League
+		</Menu.Item>
+		<Menu.Item to='/matchups' name='matchups' active={activeItem === 'matchups'} onClick={handleItemClick} style={style}>
+			Matchups
+		</Menu.Item>
+		<Menu.Item to='/rules' name='rules' active={activeItem === 'rules'} onClick={handleItemClick} style={style}>
+			Rules
+		</Menu.Item>					
+		<Menu.Item to='/profile' name='profile' active={activeItem === 'profile'} onClick={handleItemClick} style={style}>
+			Profile
+		</Menu.Item>
+		<Menu.Item to='/login' name='sign in' active={activeItem === 'sign in'} onClick={handleItemClick} style={style}>
+			Sign In
+		</Menu.Item>
+	</Menu.Menu>
+)
+
+const RoutingPaths = () => (
+	<Switch>
+		<Route exact path='/score' render={() => <ScoreForm />} />
+		<Route exact path='/scoresheet' render={() => <ScoreSheet />} />
+		<Route path='/login' render={() => <Login />} />
+		<Route path='/matchups' render={() => <HeadToHead />} />
+		<Route path='/profile' render={() => <ProfileForm userAttributes={this.state.userAttributes} preferences={this.state.preferences} handleCheckedBox={this.handleCheckedBox}/>} />
```

Comment:
Currently, `this.state.userAttributes` is undefined since it was originally in App.js. So if you try to load the Profile screen, the app crashes. Since we are currently just developing, you could just pass down userAttributes as props from App.js to MenuHeader, or just move the code from App.js into MenuHeader

Reply:
I can remove it since Ryan will be removing it in his commit for ProfileForm under iss21